framework/src/node/node.ts

```diff
@@ -340,10 +340,17 @@ export class Node {
 	public get actions() {
 		return {
 			getValidators: async (): Promise<
-				ReadonlyArray<{ address: string; nextForgingTime: number }>
+				ReadonlyArray<{
+					address: string;
+					nextForgingTime: number;
+					minActiveHeight: number;
+					isConsensusParticipant: boolean;
+				}>
 			> => {
 				const validators = await this._chain.getValidators();
 				const validatorAddresses = validators.map(v => v.address);
+				const validatorminActiveHeights = validators.map(v => v.minActiveHeight);
```

Comment:
instead of mapping one by one, it would be better to get the validators in the `for` loop below.
```
for (let i = slotInRound; i < slotInRound + this._chain.numberOfValidators; i += 1) {
  const validator = validators[i % validators.length];
  forgersInfo.push({
    nextForgingTime,
    ...validator,
    address: validator.address.toString('hex'),
});
```